CMSgov/design-system
# [WNMGDS-1244] Story Tabs Component

### Summary
Added a storybook story for the Tabs component

Went around in circles trying to decide if I should be representing submodules as stories then discovered the `submodules` arg.. 🤦 

### Added
Single story with submodules listed under "docs"

### How to test
run `yarn storybook` and see the Tab story, should be at the root level as it doesn't have variations
